osgp/protocol-adapter-mqtt/osgp-protocol-simulator-mqtt/src/main/java/org/opensmartgridplatform/simulator/protocol/mqtt/SimulatorConfig.java

```diff
@@ -9,25 +9,38 @@
 package org.opensmartgridplatform.simulator.protocol.mqtt;
 
 import java.io.IOException;
+import java.util.Properties;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
 
 @Configuration
+@Import(SimulatorClientConfig.class)
 public class SimulatorConfig {
 
   private static final Logger LOG = LoggerFactory.getLogger(SimulatorConfig.class);
 
+  @Value("${mqtt.simulator.ssl.enabled:false}")
+  private boolean sslEnabled;
+
+  @Bean
+  public Boolean isSslEnabled() {
+    return this.sslEnabled;
+  }
```

Comment:
Is this used. Looking at `SimulatorBrokerConfig` and `SimulatorClientConfig` where the `mqttBrokerProperties` and `mqttClientProperties` come from, they each have their own `boolean sslEnabled`.
Not sure why there is a default value in the broker config that is not explicitly set in the client config (though with the value being `false` the effect appears the same).